### ISSUE TITLE
HTML tags for Google Crawlers

### ISSUE THREAD

**Issue by pi0neerpat** [herecydev/gatsby-plugin-internationalization#10]
Thoughts on adding a feature to generate HTML tags describing alternate language versions of each page?

I'm not sure if its necessary to do this, but my gut tells me its probably a helpful feature.

See the HTML section in [Tell Google about localized versions of your page](https://support.google.com/webmasters/answer/189077)

**Comment by herecydev** [comment]
Yep, I spiked making the "sitemap.xml" work a while back which I think is ultimately the best solution. But this is definitely the easier one, and it'll be nice to support both models.

I'll take a look when I find some time

**Comment by herecydev** [comment]
@pi0neerpat sorry this took a little longer than expected. Do you want to check the PR and see if that configuration matches your expectation. I still need to double check the technical implementation but I _think_ it's working as expecte

**Comment by pi0neerpat** [comment]
This is awesome! Thanks so much for doing this.

I read through the link I posted above again, and the only thing that might be needed is x-default. Personally I don't think I need it, since I am automatically redirecting to the default language, based on a whitelist. So I'll leave it up to you.

> The reserved value hreflang="x-default" is used when no other language/region matches the user's browser setting. This value is optional, but recommended, as a way for you to control the page when no languages match.
